src/util/chrome.ts

```diff
@@ -34,42 +34,46 @@ export const defaultFlags:flags = {
   hideScrollbars: true,
 };
 
+export const transformChromeFlags = (flags: flags) => {
+  return _.chain(flags)
+    .pickBy((value) => value)
+    .map((_value, key) => `--${_.kebabCase(key)}`)
+    .value();
+};
+
 // Contains all the business 
 export const launch = async(flags: flags, isHost: boolean = false): Promise<chromeInstance> => {
-  const logLevel:string = process.env.DEBUG && (
+  const logLevel = process.env.DEBUG && (
```

Comment:
I removed some of these type declarations when they were being inferred correctly by typescript without them.